lib/metrics/context.js

```diff
@@ -6,7 +6,7 @@
 
 const bufferEqualConstantTime = require('buffer-equal-constant-time')
 const crypto = require('crypto')
-const HEX = require('../routes/validators').HEX_STRING
+const HEX_STRING = require('../routes/validators').HEX_STRING
```

Comment:
Tweaked this to make the validation documentation consistent. It's kind of funny how auto-generating the docs feeds back into motivations to make code more consistent. Not sure if that's a good thing or a bad thing.